### PR TITLE
Embed simple

### PR DESCRIPTION
**Pull request by cloudformdesign**
A simpler embeding example

### CODE REVIEW

File: examples/embed/polynomial/polynomial.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+'''
+This script is loosely based on the bokeh spectogram example,
+    but is much simpler:
```
Comment (bryevdv) [inline examples/embed/polynomial/polynomial.py]: suggest dedent this line, and add a blank line after the colon

File: examples/embed/polynomial/polynomial.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    https://github.com/bokeh/bokeh/tree/master/examples/embed/spectrogram
+
+This creates a simple form for generating polynomials of the form y = x^2
```
Comment (bryevdv) [inline examples/embed/polynomial/polynomial.py]: missing .

File: examples/embed/polynomial/polynomial.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+This is done using a form that has a method of GET, allowing you to share the
+    graphs you create with your friends though the link!
```
Comment (bryevdv) [inline examples/embed/polynomial/polynomial.py]: same comment about dedent

**Comment by bryevdv** [comment]
@cloudformdesign a couple of very tiny comments, but otherwise LGTM. The only other suggestion I would make is that perhaps this directory should be `examples/embed/simple` since I think the "simple" aspect is probably more relevant than the particular content of the example. Thanks!

**Comment by fpliger** [comment]
+1 on the directory suggestion. LGTM.

**Comment by cloudformdesign** [comment]
Alright, suggested changes have been made and I validated that everything still works!

**Comment by bryevdv** [comment]
Since this only touches things under `examples` this can be merged for 0.7.1 release tomorrow. I will merge as soon as TravisCI passes. Thanks @cloudformdesign !

**Comment by bryevdv** [comment]
@cloudformdesign 

my apologies I only just realized why you asked about `RESOURCES.render`, I think I was a little hasty in saying that users might never need to use this API. I said that thinking that most people embedding using `bokeh.components` would just include their own script tags to load BokehJS from CDN, but doing things as you have is not unreasonable. I did just notice, though, that you have called `components` with CDN resources, but rendered BokehJS `inline` as well. I will correct this inconsistency in a follow in PR. As you can see it doesn't prevent the example from working but it is confusing. I will also update the comment about `RESOURCES` since it should get a better reference than a GH issue link. Thanks again.

**Pull request merged**